index.js

```diff
@@ -3,4 +3,320 @@ function getGreeting (firstName, Surname) {
+console.log(divisibleExampleTwo);
+const divisibleExampleThree = isDivisible(100,5,3);
+console.log(divisibleExampleThree);
+const divisibleExampleFour = isDivisible(12,7,5);
+console.log(divisibleExampleFour);
+
+
+// Write the getBillboardPrice function that accepts two arguments:•
+// the text intended to use on the billboard•
+// the cost of a single character
+function getBillboardPrice(text, cost) {
+    return (text + ' ' + cost * 12);
+}
+
+const billboard = getBillboardPrice('Hello world!', 10); // 120
+const billboardTwo = getBillboardPrice('Hello world!', 15); // 180
+const billboardThree = getBillboardPrice('To be, or not to be', 20); // 380
+console.log(billboard);
+console.log(billboardTwo);
+console.log(billboardThree);
+
+// given an integer or a floating-point number, find its opposite.
+
+function oppositeInteger(number) {
+    return (-number);
+}
+const numberOne = oppositeInteger(1);
+console.log(numberOne);
+const numberTwo = oppositeInteger(14);
+console.log(numberTwo);
+const numberThree = oppositeInteger(8);
+console.log(numberThree);
+
+// In this simple assignment you are given a number and have to make it negative. But maybe the number is already negative?
+function negativeInteger(num) {
+    if (num > 0) {
+        return (num * -1);
+    }
+    return num
+}
+
+const numOne = negativeInteger(1);
+console.log(numOne);
+const numTwo = negativeInteger(-5);
+console.log(numTwo);
+const numThree = negativeInteger(0);
+console.log(numThree);
+const numFour = negativeInteger(0.12);
+console.log(numFour);
+
+// It's pretty straightforward. Your goal is to create a function that removes the first and last characters of a string. You're given one parameter,
+// the original string. You don't have to worry with strings with less than two characters.
+
+function removeFirstAndLastChar(str){
+    return str.slice(1,-1);
+};
+
+const removeString = removeFirstAndLastChar('Hello world')
+console.log(removeString);
+
+// Your task is to create a function that does four basic mathematical operations.
+//The function should take three arguments - operation(string/char), value1(number), value2(number).
+//     The function should return result of numbers after applying the chosen operation.
+// Examples(Operator, value1, value2) --> output
+
+function basicOp(operation, value1, value2) {
+    if (operation === '+') {
+    return value1 + value2;
+    } else if (operation === '-') {
+        return value1 - value2;
+    } else if (operation === '*') {
+        return value1 * value2;
+} else if (operation === '/') {
+        return value1 / value2;
+    }
+}
+
+const result = basicOp('+',4,7);
+console.log(result);
+const resultTwo = basicOp( '-',15,18);
+console.log(resultTwo);
+const resultThree = basicOp( '*',5,5);
+console.log(resultThree);
+const resultFour = basicOp( '/',49,7);
+console.log(resultFour);
+
+// Given a year, return the century it is in.
+
+function century(year) {
+    const x = Math.floor((year-1)/100) + 1;
```

Comment:
The `x` variable is unnecessary.

Reply:
Also, this could be a bit simpler with the use of `Math.ceil`. Try doing that instead.